packages/docusaurus-theme-classic/src/admonitions.css

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
```

Comment:
Note: this is a subset of this file: https://github.com/elviswolcott/remark-admonitions/blob/master/styles/infima.css

With proper Infima support, we can now emove a lot of CSS rules